sktime/forecasting/online_experts.py

```diff
@@ -0,0 +1,353 @@
+__author__ = ["William Zheng"]
+# __all__ = ["OnlineHedgeDoubling","OnlineHedgeIncremental", "NormalHedge"]
+
+import numpy as np
+
+
+class OnlineExperts(object):
+    """
+    Super-Class for general online expert style algorithms
+    """
+
+    def __init__(self, **kwargs):
+        pass
+
+    def _predict(self, expert_predictions):
```

Comment:
if these are meant to be abstract methods, instead of using `pass` I'd use `raise NotImplementedError()`

Reply:
Changed!